salt/elife-libraries/init.sls

```diff
@@ -230,7 +230,6 @@ mysql-user:
     mysql_user.present:
         - name: elife-libraries
         - password: elife-libraries
-        - connection_pass: {{ pillar.elife.db_root.password }}
```

Comment:
how does it work? We don't need the root password anymore to create a user because it's empty?

Reply:
https://wiki.ubuntu.com/XenialXerus/ReleaseNotes#MySQL_5.7

Reply:
it's a PITA but I can see how it's more secure *and* it will help pare back some of these `connection_pass` params. I have it working for 14.04 and 16.04 now, although this PR will have to wait until a builder PR is merged in.